Michelf/MarkdownExtra.php

```diff
@@ -1318,6 +1321,12 @@ protected function _doFencedCodeBlocks_callback($matches) {
 		$classname =& $matches[2];
 		$attrs     =& $matches[3];
 		$codeblock = $matches[4];
+
+		if ($this->code_block_content_func) {
+			$codeblock = call_user_func($this->code_block_content_func, $codeblock, $classname);
+			return "\n\n".$this->hashBlock($codeblock)."\n\n";
+		}
+
```

Comment:
Hum, I'm not sure I like the way it works, taking charge of emitting the `<pre><code>` and bypassing the extra attribute handling done below. Wouldn't it be enough if the callback only replaced the htmlspecialchars step below instead?